cmake/projects/freetype-gl/ci/build.sh

```diff
@@ -0,0 +1,36 @@
+# Install Python package 'requests'
```

Comment:
This should use the base script by default: https://hunter.readthedocs.io/en/latest/creating-new/test/ci.html#override-default-tests

Something like:
```
sudo apt-get install libgl1-mesa-dev

bash .github/workflows/ci/build.sh
```